qt/python/mantidqt/widgets/algorithmselector/widget.py

```diff
@@ -168,7 +168,8 @@ def _get_search_box_selection(self):
         i = self.search_box.findText(text)
         if i < 0:
             return None
-        return SelectedAlgorithm(name=self.search_box.currentText(), version=-1)
```

Comment:
Could I ask the reason for the temporary variable?

Reply:
This was needed for my original fix which was done at the UI level, since the changes are now made at the framework level you are right this is no longer needed.